.github/workflows/release.yml

```diff
@@ -37,3 +37,7 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Update README list of plugins and extensions
+        run: npm run update-readme.js
+      
```

Comment:
Don't forget to add the commit and push actions here.

Reply:
Quickly dropping a note here that we can just make the changesets action do the commit work for us:
```diff
    "update-unpkg-links": "gulp updateUnpkgLinks",
+   "update-readme": "node update-readme.js",
    "prepare": "husky install",
    "changeset": "changeset",
-   "changeset:version": "changeset version && npm install && npm run update-unpkg-links",
+   "changeset:version": "changeset version && npm install && npm run update-unpkg-links && npm run update-readme",
```